routing/edge_estimator.cpp

```diff
@@ -271,13 +271,12 @@ class PedestrianEstimator final : public EdgeEstimator
 
   // EdgeEstimator overrides:
   double GetUTurnPenalty(Purpose /* purpose */) const override { return 0.0 /* seconds */; }
-  // Based on: https://confluence.mail.ru/display/MAPSME/Ferries
   double GetFerryLandingPenalty(Purpose purpose) const override
   {
     switch (purpose)
     {
-    case Purpose::Weight: return 20.0 * 60.0;  // seconds
-    case Purpose::ETA: return 8.0 * 60.0;      // seconds
+    case Purpose::Weight: return 10 * 60;   // seconds
+    case Purpose::ETA: return 8 * 60;       // seconds
```

Comment:
We can start manually building a list of ferries' load times/duration and inject this info in the generator. Maybe like a config file, that can be easily updated by the community.